wsgi/odyssy/people/templates/people.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'basic/base.html' %}
+{% load static %}
+{% block body %}
+    <div class="center-block people">
+        <h2>{{ status }}</h2>
+        <ul class="center-block list-group">
+            {% for people in people_list %}
+                <li class="list-group-item">
+                    {% if people.slug %}
+                        <h3><a href="{{ people.get_url }}">{{ people.name }}</a></h3>
+                    {% else %}
+                        <h3>{{ people.name }}</h3>
+                    {% endif %}
+                    <div class="row">
+                        <div class="col-sm-4 col-md-4 col-lg-4">
+                            <img width="150px" height="150px" src="{% static people.photo %}" alt="">
```

Comment:
@Monal5031 That's because of the deleted media folder, everything alright please check again.